test/ArrayFire/ArithSpec.hs

```diff
@@ -22,23 +22,23 @@ spec =
     it "Should subtract two scalar arrays" $ do
       scalar @Int 4 - 2 `shouldBe` 2
     it "Should multiply two scalar arrays" $ do
-      scalar @Double 4 `mul` 2 `shouldBe` 8
+      scalar @Double 4 `mul` 2 `shouldBeEps` 8
     it "Should divide two scalar arrays" $ do
-      div @Double 8 2 `shouldBe` 4
+      div @Double 8 2 `shouldBeEps` 4
     it "Should add two matrices" $ do
       matrix @Int (2,2) [[1,1],[1,1]] + matrix @Int (2,2) [[1,1],[1,1]]
         `shouldBe`
            matrix @Int (2,2) [[2,2],[2,2]]
     it "Should take cubed root" $ do
-      3 `shouldBe` cbrt @Double 27
+      3 `shouldBeEps` cbrt @Double 27
     it "Should take square root" $ do
-      2 `shouldBe` sqrt @Double 4
+      2 `shouldBeEps` sqrt @Double 4
     it "Should lt Array" $ do
-      2 `A.lt` (3 :: Array Double) `shouldBe` True
+      2 `A.lt` (3 :: Array Double) `shouldBeEps` True
```

Comment:
This is not the solution. `lt` returns an array of doubles, and you compare it with a boolean. You could do:

```
      2 `A.lt` (3 :: Array Double) `shouldBe` 1
```

Reply:
ah, yeah, that shouldn't have been changed there.

Reply:
But maybe this should wait until after #24

Reply:
@chessai read my comment more carefully, it is not just about `shouldBeEps`.

Reply:
yeah, sorry. i'm going to remove these for now.